src/main/java/com/lobo/onThisDay/service/BirthService.java

```diff
@@ -0,0 +1,34 @@
+package com.lobo.onThisDay.service;
+
+import com.lobo.onThisDay.model.Person;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.time.MonthDay;
+import java.util.List;
+
+/**
+ *
+ * Service que conterá a logica para trazer os dados de nasicmento.
+ *
+ * @author Gustavo Lobo
+ */
+@Service
+public class BirthService {
+
+    @Autowired
+    private OnThisDayScrapperService onThisDayScrapperService;
+
+    /**
+     * Esse metodo temporariamente ainda busca apenas as pessoas de destaque nas paginas de nascimento, será incrementado
+     * futuramente para buscar a todos.
+     * @param date
+     * @return
+     * @throws IOException
+     */
+    public List<Person> getPersonsBornIn(MonthDay date) throws IOException {
+        return onThisDayScrapperService.getHighlightPersonsBornIn(date);
```

Comment:
Pelo que entendi esse cara vai encapusular diversas logicas dentro dele, seria isso? Se for é melhor repensar um pouco e mudar a abordagem, pode usar um strategy ou algo do gênero para decidir que tipo de dado vc vai recuperar.

Reply:
A ideia desse metodo é conseguir fazer a extração de todas as pessoas de uma determinada data, em ambas a fonte dos dados seriam as mesmas, o que mudaria é que eu teria dois metodos dedicados, onde a função de um seria buscar as pessoas que vem na seção de destaques da pagina (que é o metodo atual do getHiglight), e outro metodo que buscaria a partir de varios `<li>` numa outra div da mesma pagina, ficaria algo assim:

```
// Inicializando a lista e buscando as pessoas de destaque.
List<Person> pessoas = onThisDayScrapperService.getHighlightPersonsBornIn(date);

// Incrementando a lista com metodo que será implementado.
pessoas.addAll(onThisDayScrapperService.getPersonsFromLITagsBornIn(date));

return pessoas;
```